projects/storefrontlib/src/cms-components/myaccount/update-email/update-email-form/update-email-form.component.ts

```diff
@@ -1,21 +1,12 @@
 import { Component, EventEmitter, Output } from '@angular/core';
-import {
-  AbstractControl,
-  FormBuilder,
-  FormGroup,
-  ValidationErrors,
-  Validators,
-} from '@angular/forms';
-import { FormUtils } from '../../../../shared/utils/forms/form-utils';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
 import { CustomFormValidators } from '../../../../shared/utils/validators/custom-form-validators';
 
 @Component({
   selector: 'cx-update-email-form',
   templateUrl: './update-email-form.component.html',
 })
 export class UpdateEmailFormComponent {
```

Comment:
Breaking changes to document:
- removed submited, isEmailConfirmNotValid and isNotValid

Reply:
Done